### DISCUSSION TITLE
Not working on Windows system | Nuclei 3.2.9 Latest

### DISCUSSION THREAD

**Discussion by rambothehunter** [projectdiscovery/nuclei#5391] [Q&A]
I have installed the nuclei 3.2.9 on windows 10 enterprise/ 64 bit operating system. 
When I ran the software, I received the below result show in the image. 

Command used: nuclei -u example.com
____  __  _______/ /__  (_)
  / __ \/ / / / ___/ / _ \/ /
 / / / / /_/ / /__/ /  __/ /
/_/ /_/\__,_/\___/_/\___/_/   v3.2.9

                [projectdiscovery.io](http://projectdiscovery.io/)

[INF] Your current nuclei-templates v9.9.0 are outdated. Latest is v9.9.0
[ERR] Could not parse nuclei-ignore file: yaml: line 9: mapping values are not allowed in this context
[WRN] Found 160 templates with runtime error (use -validate flag for further examination)
[INF] Current nuclei version: v3.2.9 (outdated)
[INF] Current nuclei-templates version: v9.9.0 (outdated)
[WRN] Scan results upload to cloud is disabled.
[INF] New templates added in latest release: 164
[INF] Templates loaded for current scan: 8248
[INF] Executing 8248 signed templates from projectdiscovery/nuclei-templates
[INF] Targets loaded for current scan: 1
[INF] Running httpx on input host
[INF] Found 0 URL from httpx
[INF] Templates clustered: 1552 (Reduced 1461 Requests)
[INF] Skipped [example.com:80](http://example.com/) from target list as found unresponsive 30 times
[INF] No results found. Better luck next time!

![Uploading IMG_20240709_130825.jpg…]()

Please help me out that what should I do so that it will work fine for me.

**Comment by dogancanbakir** [comment]
This discussion closed automatically due to inactivity. Feel free to reopen or start new if still relevant.
